src/cli/export.ts

```diff
@@ -107,7 +107,7 @@ export function exportLump(options: {
         const playpal: WADPalette = WADPalette.from(lump);
         const data: Buffer = playpal.getPixelDataRGBA();
         const png = UPNG.encode(
-            [data.buffer], 16, 16 * playpal.getPaletteCount(), 0
+            [data.buffer as ArrayBuffer], 16, 16 * playpal.getPaletteCount(), 0
```

Comment:
Were you having problems with this without `as ArrayBuffer`?

Reply:
This gives me a Typescript compilation error if I use ES2017 as the target, which is necessary for using String.padStart/padEnd

Reply:
Oh, I hadn't realized padStart/padEnd were ES2017.

Chrome and Firefox on my laptop both support the padding methods, so I think it's probably safe from a support standpoint to target ES2017? Let's keep this change and if it causes problems later I can think about reverting and adding a pad helper instead or something.